File: tests/e2e/fixtures/go-gotestsum/calc_test.go
package calc

import "testing"

func TestAdultAt18(t *testing.T) {
	if !IsAdult(18) {
		t.Fatal("18 should be an adult")
	}
}

func TestAdultAt30(t *testing.T) {
	if !IsAdult(30) {
		t.Fatal("30 should be an adult")
	}
}

File: tests/e2e/fixtures/php-phpunit/tests/CalcTest.php
<?php

use PHPUnit\Framework\TestCase;

require_once __DIR__ . '/../src/Calc.php';

final class CalcTest extends TestCase
{
    public function testAdultAt18(): void
    {
        $this->assertTrue(Calc::isAdult(18));
    }

    public function testAdultAt30(): void
    {
        $this->assertTrue(Calc::isAdult(30));
    }
}
